next.md

```diff
@@ -116,3 +116,60 @@ export const Container = styled.div`
   background-image: url(${background});
 `;
 ```
+
+#### Next.js with styled components
+
+- [Example app with styled-components](https://github.com/vercel/next.js/tree/canary/examples/with-styled-components3)
+
+or you can install it yourself.
+
+```
+yarn add styled-components
+yarn add babel-plugin-styled-components
+```
+
+Then add 2 files. .babelrc to the root folder. and \_document.js to pages
+
+- .babelrc
+
+```
+{
+  "presets": ["next/babel"],
+  "plugins": ["styled-components"]
+}
+```
+
+- \_document.js
+
+```jsx
```

Comment:
add code to spoiler https://gist.github.com/jbsulli/03df3cdce94ee97937ebda0ffef28287

Reply:
previous spoiler is also not displaying properly, fix please